newproject01/newproject01/q2.c

```diff
@@ -0,0 +1,39 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#include <string.h>
+#define N 51
+char letterAtIndex(char*, int);
```

Comment:
add spacing between this function declaration and the define above and main below.